### ISSUE TITLE
Parameters vs Properties

### ISSUE THREAD

**Issue by hylkevds** [opengeospatial/sensorthings#176]
The most asked question: Why does Observation have `parameters` even though all other classes have `properties`?

Because OMS calls it parameters... But OMS doesn't have properties. So how about we just rename parameters to properties and document it as such?

**Comment by liangsteve** [comment]
Agree.  (I guess you meant "Because **O&M v2.0** calls it parameters..."?)

**Comment by hylkevds** [comment]
Yes, of course.
Though OMS also calls it parameters :)
